enterprise/frontend/src/metabase-enterprise/sandboxes/components/LoginAttributesWidget.tsx

```diff
@@ -24,9 +28,14 @@ export const LoginAttributesWidget = ({
     }
   };
 
+  const { values } = useFormikContext<{ tenant_id: number }>();
+
+  const { data } = useGetTenantQuery(values.tenant_id ?? skipToken);
+
   return (
     <FormField className={className} style={style} title={title}>
       <MappingEditor
+        disabledValues={getDisabledTenantUserAttribute(data)}
```

Comment:
I don't really love how this was done, but basically we want to show the `@tenant.slug` attribute if it's applicable. Instead of passing props in, I'm taking advantage of the fact that there is a form context I can look at and build the information that way.